httpcore5/src/main/java/org/apache/hc/core5/http/impl/io/DefaultBHttpClientConnectionFactory.java

```diff
@@ -140,12 +140,12 @@ public Builder charCodingConfig(final CharCodingConfig charCodingConfig) {
             return this;
         }
 
-        public Builder incomingContentStrategy(final ContentLengthStrategy incomingContentStrategy) {
+        public Builder incomingContentLengthStrategy(final ContentLengthStrategy incomingContentStrategy) {
```

Comment:
You forgot to rename the instance field names and argument name. They are still misleading.

Reply:
Updated and resolved conflicts with the new responseOutOfOrderStrategy field.